### PR TITLE
fix: wallet name can not be same

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 1]
```diff
@@ -77,6 +77,13 @@ class WalletsController {
     changeAddressNumber: number
   }): ChannelResponse<WalletData> => {
     try {
+      const id = WalletsController.service.getAll().findIndex(wallet => wallet.name === name)
+      if (id !== -1) {
+        return {
+          status: ResponseCode.Fail,
+          msg: 'Same name',
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/wallets.ts]:
```suggestion
          msg: 'Wallet name existed',
```

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 2]
```diff
@@ -77,6 +77,13 @@ class WalletsController {
     changeAddressNumber: number
   }): ChannelResponse<WalletData> => {
     try {
+      const id = WalletsController.service.getAll().findIndex(wallet => wallet.name === name)
```
Comment (classicalliu) [inline packages/neuron-wallet/src/controllers/wallets.ts]: Check this in data level is better, data level should ensure wallet name not duplicated.

Comment (BaojunCZ) [reply packages/neuron-wallet/src/controllers/wallets.ts]: I will move to wallet services.Store level is atom.

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets.ts]: If service throw an error, does controller should catch it and return status and msg ? I'm not sure, @Keith-CY please check it, 3Q

Comment (Keith-CY) [reply packages/neuron-wallet/src/controllers/wallets.ts]: It will be caught in walletController.create

**Pull request merged**